example-config.yaml

```diff
@@ -2,10 +2,26 @@ stormpath:
   application:
     href: null
     name: null
-    
+
   web:
+
     basePath: null
```

Comment:
What is this used for again?  How is it different than `baseDomain`?

Why not have a `baseUrl` that represents both?  Just trying to understand the relevance between these 3 concepts...